cms/djangoapps/contentstore/rest_api/v1/tests/test_views.py

```diff
@@ -182,6 +183,42 @@ def test_update_exam_settings_200(self):
         assert updated.proctoring_provider == 'proctortrack'
         assert updated.proctoring_escalation_email == 'foo@bar.com'
 
+    @override_settings(
+        PROCTORING_BACKENDS={
+            'DEFAULT': 'null',
+            'test_proctoring_provider': {}
+        },
+    )
+    @override_waffle_flag(ENABLE_PROCTORING_PROVIDER_OVERRIDES, True)
+    def test_update_exam_settings_200_no_escalation_email(self):
+        """ escalation email may be blank if not required by the provider """
+        self.client.login(username=self.global_staff.username, password=self.password)
+        data = self.get_request_data(
+            enable_proctored_exams=True,
+            proctoring_provider='test_proctoring_provider',
+            proctoring_escalation_email=''
+        )
+        response = self.make_request(data=data)
+
+        # response is correct
+        assert response.status_code == status.HTTP_200_OK
+        self.maxDiff = None
```

Comment:
What does this do?

Reply:
Looks like a copy and paste left over. Please remove.

Reply:
oh whoops. pytest debugging, I'll remove it